apitools/base/protorpclite/protojson.py

```diff
@@ -206,6 +206,7 @@ def decode_message(self, message_type, encoded_message):
           ValueError: If encoded_message is not valid JSON.
           messages.ValidationError if merged message is not initialized.
         """
+        encoded_message = six.ensure_str(encoded_message)
```

Comment:
We'll need to change the `setup.py` file to require a newer minimum version of `six` for this, right? (I think the `ensure_<foo>` stuff was introduced in 1.12, and we currently only require >=1.9).

Reply:
You're right! Updated to >= 1.12